python/kserve/kserve/model.py

```diff
@@ -346,6 +367,7 @@ async def predict(
         self,
         payload: Union[Dict, InferRequest, ModelInferRequest],
         headers: Dict[str, str] = None,
+        response_header: Dict[str, str] = None,
```

Comment:
```suggestion
        response_headers: Dict[str, str] = None,
```

Reply:
what if user updates to use the new kserve version but does not change the predict signature in their code?